kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java

```diff
@@ -122,11 +122,12 @@ public void deleteAccount(Account account, String passphrase) throws DeleteAccou
         }
     }
 
-    public void wipeoutAccount() {
+    public void wipeoutAccount() throws EthereumClientException {
         File keystoreDir = new File(getKeyStoreBasePath().toString());
         if (keystoreDir.exists()) {
             deleteRecursive(keystoreDir);
         }
+        initKeyStore();
```

Comment:
why is this needed?

Reply:
I just added it otherwise geth returns the last account even though we've wiped out the directory